### DISCUSSION TITLE
请问有c++调用paddlenlp接口吗

### DISCUSSION THREAD

**Discussion by feassh** [PaddlePaddle/PaddleNLP#2851] [Q&A]
没找到c++调用方式

**Comment by wj-Mcat** [comment]
There is no c++ API in paddlenlp, but paddle. Can you describe what do you want to do with paddlenlp and c++ ? Maybe we can help you complete it in some way.

**Reply by feassh** [reply]
我想在golang中使用paddlenlp的语义相似度功能，但是官方没有go的nlp api，如果有c/c++的话，就能在golang中使用了

**Reply by wj-Mcat** [reply]
你这个思路暂时没办法实现，不过你可以通过serving的方式来实现：部署语义相似度服务，然后再通过http 来调用都是可以的

**Reply by feassh** [reply]
好的，谢谢

**Reply by wj-Mcat** [reply]
Please feel free to tell us more if you have any issues in your next works on paddlenlp. Glad to know you have got the solution.
